src/eventloop.rs

```diff
@@ -486,6 +487,7 @@ where
     }
 }
 
+#[must_use = "Event loop is deleted when handle is dropped"]
```

Comment:
I wonder why you need that one given that it is an internal type? Just for the esp-idf-svc code itself, or am I missing something?

Reply:
I was just about to open the PR with the `EspSubscription` annoation and I thought "I'll quickly check if there's any other Drop impls" and so stuck one on this - didn't even notice it wasn't pub.

I've pushed 717c0fb to remove this one :+1: